Calculator/Calculator/Controller/CalculatorViewController.swift

```diff
@@ -28,25 +28,25 @@ class CalculatorViewController: UIViewController {
     
     override func viewDidLoad() {
         super.viewDidLoad()
-        inputNumberLabel.text = "0"
-        inputOperatorLabel.text = ""
+        inputNumberLabel.text = CalculatorNamespace.Zero
+        inputOperatorLabel.text = CalculatorNamespace.Empty
         formulaListStackView.arrangedSubviews.forEach{ $0.removeFromSuperview() }
     }
     
     @IBAction func tapNumbersButton(_ sender: UIButton) {
-        if isComputable {
-            guard let inputNumberText = sender.titleLabel?.text,
-                  let numberLabelText = inputNumberLabel.text,
-                  let operatorLabelText = inputOperatorLabel.text else { return }
-            
-            if numberLabelText.count < 20 {
-                inputOperatorLabel.text = (formulaListStackView.subviews.isEmpty) ? ("") : operatorLabelText
-                if numberLabelText == "0" {
-                    inputNumberLabel.text = inputNumberText
-                } else {
-                    let formattedNumberText = numberLabelText.replacingOccurrences(of: ",", with: "") + inputNumberText
-                    inputNumberLabel.text = calculatorNumberFormatter.string(for: Double(formattedNumberText))
-                }
+        guard let inputNumberText = sender.titleLabel?.text,
+              let numberLabelText = inputNumberLabel.text else { return }
+        
+        let numberLabelTextWithoutComma = numberLabelText.replacingOccurrences(of: CalculatorNamespace.Comma,
+                                                                               with: CalculatorNamespace.Empty)
+        
+        if isComputable, numberLabelText.count < 19 {
```

Comment:
1. 해당 조건문도 guard문을 사용할 수 있을 것 같아보여요.
```suggestion
        guard let inputNumberText = sender.titleLabel?.text,
              let numberLabelText = inputNumberLabel.text,
              isComputable, 
              numberLabelText.count < 19  else { return }
              
        let numberLabelTextWithoutComma = numberLabelText.replacingOccurrences(
              of: CalculatorNamespace.Comma,
              with: CalculatorNamespace.Empty
        )
```

2. inputNumberText, inputNumberLabel은 뜻하는 바가 전혀 다른 두가지입니다. 하지만 이 네이밍이 겹쳐 헷갈리는 것 같아요. 좀 더 명확한 네이밍을 하면 더 좋을 것 같습니다.

Reply:
코멘트 참고하여 `guard`문으로 수정하였습니다! 꼼꼼한 리뷰 감사합니다😆
`inputNumberText`와 `currentNumberLabelText`로 서로 구분하였습니다!